### ISSUE TITLE
Circulars: split submitter field into multiple fields for improved Lucene search compatibility

### ISSUE THREAD

**Issue by tylerbarna** [nasa-gcn/gcn.nasa.gov#2399]
# Description

In order to implement searching by submitter through lucene style search queries, it will be necessary to change the way we encode the submitter information. The current structure of the submitter field is along the lines of:

`"[submitter name] at [institution] <[submitter email]>"`

Lucene style search queries would benefit from splitting this into multiple fields, such as
- submitter name
- submitter institution
- submitter contact

In particular, splitting out the submitter name into its own field would allow for searching for specific submitters.

# Acceptance criteria

- [ ] Schema is changed such that submitter field separates out the name of submitter and institution/contact information
- [ ] Existing Circulars are modified to match the new schema
- [ ] Opensearch is updated to account for this change
- [ ] (optional) the submitter name is split by first and last name. This would unblock #299, but it is also a significant undertaking that will require verifying the first and last names of existing submitters

**Comment by lpsinger** [comment]
Duplicate of #893.
